ledger/sandbox-on-x/src/main/scala/com/daml/ledger/sandbox/bridge/ConflictCheckingLedgerBridge.scala

```diff
@@ -132,22 +133,22 @@ private[sandbox] class ConflictCheckingLedgerBridge(
         Timed
           .future(
             bridgeMetrics.Stages.conflictCheckWithCommitted,
-            checkTimeModel(originalSubmission) match {
```

Comment:
`checkTimeModel` is performed in `sequence` in order to have access to the record time.